integration_tests/e2e/prisonerChangeIncentiveLevelDetails.cy.ts

```diff
@@ -0,0 +1,122 @@
+import Page from '../pages/page'
+import PrisonerChangeIncentiveLevelDetailsPage from '../pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsPage'
+import PrisonerChangeIncentiveLevelConfirmationPage from '../pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsConfirmationPage'
+
+const IncentiveSummary = {
+  bookingId: -1,
+  iepDate: '2017-08-15',
+  iepTime: '2017-08-15T16:04:35',
+  iepLevel: 'Basic',
+  daysSinceReview: 1868,
+  nextReviewDate: '2022-09-26',
+  iepDetails: [
+    {
+      bookingId: -1,
+      iepDate: '2017-08-15',
+      iepTime: '2017-08-15T16:04:35',
+      agencyId: 'MDI',
+      iepLevel: 'Basic',
+      userId: 'INCENTIVES_API',
+      comments: 'INCENTIVES_API_COMMENT',
+    },
+  ],
+}
+context('Prisoner change incentive level details', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubFallbackHeaderAndFooter')
+    cy.task('stubNomisUserRolesGetCaseloads')
+    cy.task('stubManageUser')
+    cy.task('stubNomisUserRolesApiUserCaseloads')
+    cy.task('stubPrisonApiLocations')
+    cy.task('stubPrisonIncentiveLevels')
+
+    const roles = ['ROLE_MAINTAIN_IEP']
+    cy.task('stubSignIn', { roles })
+    cy.navigateToChangePrisonerIncentiveLevelDetails().then(result => {
+      return result
+    })
+  })
+
+  context('When user has CORRECT role', () => {
+    it('should display prisoner name and current level', () => {
+      const page = Page.verifyOnPage(PrisonerChangeIncentiveLevelDetailsPage)
+      page.pageHeading.should('contain.text', 'Record John Smith’s incentive level')
```

Comment:
this is actually already tested by `Page.verifyOnPage(PrisonerChangeIncentiveLevelDetailsPage)` – that's what the text inside the Page’s `super` call is for, to check that the h1 contains it.
so we don't need this line nor the `pageHeading` getter.